core-services/ci-secret-bootstrap/_config.yaml

```diff
@@ -1413,3 +1413,11 @@
     - cluster: app.ci
       namespace: ci
       name: release-controller-signature-signer
+- from:
+    oauth:
+      bw_item: gcsweb-private-oauth
```

Comment:
what oauth token is this? GCS? Github?

Reply:
GCS oauth. To be more specific, this token was generated from a specific service account with limited permissions. (read-only GCS storage)

Reply:
Can you then name everything (bitward entry, secret, mount) to reflect that this is a GCP SA and what name it has? The fact that its an oauth token tells me nothing about it.

Reply:
Its not a service account. Its the OAuth only. But I will change the name to be more specific. Thanks for suggesting this.